ludwig/schema/llms/base_model.py

```diff
@@ -45,17 +46,20 @@ def validate(model_name: str):
         """Validates and upgrades the given model name to its full path, if applicable.
 
         If the name exists in `MODEL_PRESETS`, returns the corresponding value from the dict; otherwise checks if the
-        given name (which should be a full path) exists in the transformers library.
+        given name (which should be a full path) exists locally or in the transformers library.
         """
         if isinstance(model_name, str):
             if model_name in MODEL_PRESETS:
                 return MODEL_PRESETS[model_name]
+            if os.path.isdir(model_name):
+                return model_name
             try:
                 AutoConfig.from_pretrained(model_name)
                 return model_name
             except OSError:
                 raise ConfigValidationError(
                     f"Specified base model `{model_name}` is not a valid pretrained CausalLM listed on huggingface. "
+                    "or a valid directory containing a pretrained CausalLM."
```

Comment:
nit: Probably want to remove the full stop from the previous line. 

Also, just to people are absolutely clear, should we say `or a valid local directory containing the weights for a pretrained CausalLM from huggingface`? wdyt?

Reply:
That makes sense to me